LiskHQ/lisk-sdk
# Add missing apply penalty from LIP-0004 - Closes #3672

### What was the problem?
Apply penalty was not completely applied to all the RPC endpoints.

### How did I solve it?
- Add apply penalty on the schema errors.
- Expose peerId on the request endpoints

### How to manually test it?
- run tests
- observe behavior with 2 nodes

### Review checklist

- [ ] The PR resolves #3672
